app/models/popup_notification.rb

```diff
@@ -0,0 +1,4 @@
+class PopupNotification < ActiveRecord::Base
```

Comment:
Either make this Notification::Popup (which I think was what you were asking...) or make this `NotificationPopup` On hindsight  I think Notification::Popup is better.